integration/src/main/java/scripts/utils/GcsBucketAccessTester.java

```diff
@@ -154,17 +265,34 @@ public interface TestFunction {
   }
 
   private boolean doWithOptionalWait(TestFunction function) throws Exception {
```

Comment:
can we get rid of the field needToWait and pass this in through the param? Similar to `checkAccessWorker`

Reply:
I don't understand the suggestion.

Reply:
doWithOptionalWait(TestFunction function, boolean wait)

Reply:
I don't think I want to do that.

Reply:
why not? it's only used by `checkAccessWorker` and `needToWait` is only set in `checkAccessWorker`.

Reply:
Because it is a style change for a blocking PR

Reply:
I fixed this and the annotation of /wait=/